povm_toolbox/library/randomized_projective_measurements.py

```diff
@@ -251,6 +256,17 @@ def to_sampler_pub(
             ).reshape(  # Reshape to match the shape of ``pvm_idx``.
                 (*circuit_binding.shape, num_batches)
             )
+            # If the twirling option is turned on we double the number of PVMs
+            # because each PVM can be twirled. The encoding works as follows :
+            # `pvm_idx % self._n_PVMS` is the index of the PVM used and
+            # `pvm_idx // self._n_PVMS` indicates if the PVM has been twirled.
```

Comment:
In other words, for the example of `ClassicalShadows` we have:
```python
n_PVMs = 3
# pvm_idx == 0 -> untwirled X
# pvm_idx == 1 -> untwirled Y
# pvm_idx == 2 -> untwirled Z
# pvm_idx == 3 -> twirled X
# pvm_idx == 4 -> twirled Y
# pvm_idx == 5-> twirled Z
```
correct?

If so, I like it, because the `pvm_idx` from 0 to 2 keeps the same interpretation regardless of whether or not we twirl :+1:

Reply:
Yes exactly ! I can add this example in the docstrings 👍